### PR TITLE
Cleanups in Lua Api doc

### PR DESCRIPTION
**Pull request by enjoy-binbin**

### CODE REVIEW

File: docs/manual/programmability/lua-api.md [hunk 1]
```diff
@@ -133,16 +133,16 @@ return redis.call('ECHO', 'Echo, echo... eco... o...')
 ```
 
 If and when `redis.call()` triggers a runtime exception, the raw exception is raised back to the user as an error, automatically.
-Therefore, attempting to execute the following ephemeral script will fail and generate a runtime exception because `ECHO` accepts exactly zero or one argument:
+Therefore, attempting to execute the following ephemeral script will fail and generate a runtime exception because `ECHO` accepts exactly one argument:
 
 ```lua
 redis> EVAL "return redis.call('ECHO', 'Echo,', 'echo... ', 'eco... ', 'o...')" 0
-(error) ERR Error running script (call to b0345693f4b77517a711221050e76d24ae60b7f7): @user_script:1: @user_script: 1: Wrong number of args calling Redis command from script
+(error) ERR Wrong number of args calling Redis command from script script: b0345693f4b77517a711221050e76d24ae60b7f7, on @user_script:1.
```
Comment (enjoy-binbin) [inline docs/manual/programmability/lua-api.md:140]: this change is due to https://github.com/redis/redis/pull/10329

File: docs/manual/programmability/lua-api.md [hunk 2]
```diff
@@ -331,7 +331,7 @@ It then picks five random elements (`SRANDMEMBER`) from the intersection and sto
 Finally, before returning, it deletes the temporary key that stores the intersection of the two source sets.
 
 In this case, only the new set with its five randomly-chosen elements needs to be replicated.
-Replicating the `SUNIONSTORE` command and the `DEL'ition of the temporary key is unnecessary and wasteful.
+Replicating the `SUNIONSTORE` command and the `DEL'ition` of the temporary key is unnecessary and wasteful.
```
Comment (itamarhaber) [inline docs/manual/programmability/lua-api.md:334]:
```suggestion
Replicating the `SUNIONSTORE` command and the `DEL`ition of the temporary key is unnecessary and wasteful.
```

**Review by itamarhaber — APPROVED** [review]
@enjoy-binbin thank you!

**Comment by enjoy-binbin** [comment]
i pushed a commit to fix the spellcheck

**Pull request merged**